### PR TITLE
Feature/start masternode

### PR DESCRIPTION
**Pull request by tsimafeisetchinava000**
Add starting masternode(HOT/HOT) feature.

### CODE REVIEW

**Review by a-ok123 — DISMISSED** [review]

File: cmd/app.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+	// defaultConfigFile = ""
+)
+
+// AppWriter writer for logging
+var AppWriter io.Writer
+
+// NewApp inits a new command line interface.
+func NewApp() *cli.App {
+
```
Comment (alonlong) [inline cmd/app.go:28]: it's better to remove the empty line

File: utils/sshclient.go [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+		_type:      scriptFile,
+		client:     c.client,
+		scriptFile: fname,
+	}
+}
+
+// A RemoteScript represents script that can be run remotely.
+type RemoteScript struct {
+	client     *ssh.Client
+	_type      remoteScriptType
```
Comment (alonlong) [inline utils/sshclient.go:153]: can u help to change the field name? like scriptType

Comment (tsimafeisetchinava000) [reply utils/sshclient.go:153]: I see.

File: utils/sshclient.go [hunk 3]
```diff
@@ -150,7 +150,7 @@ func (c *Client) ScriptFile(fname string) *RemoteScript {
 // A RemoteScript represents script that can be run remotely.
 type RemoteScript struct {
 	client     *ssh.Client
-	_type      remoteScriptType
```
Comment (alonlong) [inline utils/sshclient.go]: sorry, i should say more clearly, i mean the name _type should be changed

**Review by a-ok123 — DISMISSED** [review]

File: utils/sshclient.go (hunk 3 above)
Comment (tsimafeisetchinava000) [reply utils/sshclient.go]: I see.
Thank you.

**Review by a-ok123 — APPROVED** [review]
You put everything into one PR, this is quite hard to review, so let's just merge it and we will test how the tools is working

**Review by a-ok123 — APPROVED** [review]

**Pull request merged**